tests/clickhouse-test

```diff
@@ -1173,6 +1173,22 @@ class TestCase:
             description_full += result.reason.value
 
         description_full += result.description
+
+        if BuildFlags.SANITIZE_COVERAGE in args.build_flags:
```

Comment:
Is `args` defined here?

*after 5 minutes of jumping around the code*

Oh well.. it's very obfuscated, where it's global, local, and where it has a completely different meaning than anywhere else in the file. But yes, it's definied globally in `if __name__ == "main"`